configure

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
 
+import sys
+if sys.version_info >= (3, 0):
+  sys.stdout.write("Python 3.x is not compatible at this time\n")
```

Comment:
@silverwind: I guess my sentiment was that that code path likely would be unreachable.